x-pack/test/security_solution_api_integration/test_suites/detections_response/rules_management/rule_creation/basic_license_essentials_tier/create_ml_rules_privileges.ts

```diff
@@ -49,20 +49,25 @@ export default ({ getService }: FtrProviderContext) => {
         await deleteAllRules(supertest, log);
       });
 
-      it('@ess should give a 403 when trying to create a single Machine Learning rule since the license is basic', async () => {
-        const { body } = await supertest
-          .post(DETECTION_ENGINE_RULES_URL)
-          .set('kbn-xsrf', 'true')
-          .set(ELASTIC_HTTP_VERSION_HEADER, '2023-10-31')
-          .send(getSimpleMlRule())
-          .expect(403);
+      describe('@ess', function () {
+        /* Wrapped in `describe` block, because `this.tags` only works in `describe` blocks */
```

Comment:
I've added comments in https://github.com/elastic/kibana/pull/192053/commits/4a9f635b0d866ff4a4eb0cf7cfd6add0eeef241d to explain why `describe`s are needed